src/handlers/call-handler.ts

```diff
@@ -1,28 +1,46 @@
-import { isFunction } from '../utils/types'
-import { EventBus, ExternalCallHandler, ICallHandler } from '../types'
+import { EventBus, ExternalCallHandler } from '../types'
+import { WrappingContext } from '../utils/wrapping'
 
-const _noOp = () => undefined
+interface WrappedExternalCallHandler {
+  ajaxGet: (
+    url: string,
+    onSuccess: (responseText: string, response: unknown) => void,
```

Comment:
```suggestion
    onSuccess: (responseText: string, response ?: unknown) => void,
``````

Reply:
no need, the callback will always be provided with the argument

Reply:
but when it is called it uses only the first param
https://github.com/LiveIntent/live-connect/blob/master/src/pixel/sender.ts#L57

Reply:
Using only one is fine. The questionmark would signal that the function is only provided 1 argument sometimes, even if it asks for 2.